region/brinstar/pink/Big Pink.json

```diff
@@ -1955,6 +1959,28 @@
         "Refreeze the Zeb below the crumble blocks and ice clip up to reach the door."
       ]
     },
+    {
+      "link": [13, 5],
+      "name": "Big Pink Blind Zeb Ice Clip (Preserve Flash Suit)",
+      "notable": true,
+      "requires": [
+        "canStaggeredWalljump",
+        "canCameraManip",
+        "canOffScreenMovement",
+        "Wave",
+        "canInsaneJump",
+        "h_canHighPixelIceClip"
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Raise a Zeb to be just below the crumble blocks and blindly freeze it to set up an ice clip to reach the Power Bomb room exit door.",
+        "Get a bug from the pipe to spawn facing left and freeze it while moving to the ledge below the Hopper room door.",
+        "Go above, then blindly freeze it with a diagonal downward shot at a pixel perfect height.",
```

Comment:
A standing shot followed by a jump? from where the normal setup position ( below wave gate door) is should work too.  Just a smaller frame window.  Refreezing by the crumbles would work the same.  but I haven't tried it.

Reply:
This felt harder to me, but I didnt try for too long. If you shoot before you jump, itll be frozen too low. So you might either need a second blind freeze or a high jump with diagonal shot (probably wont work).

Reply:
I haven't gotten it yet, but the height looks possible from a crouch shot into (step forward) spin jump.  Then you could do it with spazer, and keep the flash suit with an unmorph upshot on the refreeze.